modules/static/opencensus/opencensus/initref

```diff
@@ -1 +1 @@
-d5d80953a8c2ff4633087af6933cd152678434bb
+v0.0.2
```

Comment:
This is the the closest tag:

```
➜  opencensus-proto git:(d5d8095) git describe --all d5d80953a8c2ff4633087af6933cd152678434bb
tags/v0.0.2-55-gd5d8095
```

Here are all the releases: https://github.com/census-instrumentation/opencensus-proto/releases

This repo is archived and we don't expect any more updates.